hybrasyl/Objects/User.cs

```diff
@@ -1073,6 +1070,43 @@ public override void SendLocation()
             x04.WriteUInt16(11);
             x04.WriteUInt16(11);
             Enqueue(x04);
+
+            var doors = GetDoorsInView(GetViewport());
+
+            if(doors.Count > 0)
+            {
+                foreach(var door in doors)
+                {
+                    SendDoorUpdate(door.Item1, door.Item2, Map.Doors[door].Closed, Map.Doors[door].IsLeftRight);
+                }
+            }
+
+        }
+
+        public List<Tuple<byte,byte>> GetDoorsInView(Rectangle viewPort)

```

Comment:
i would prefer this be more obvious? `GetDoorCoordinatesInView` since it isn't actually returning door objects?